<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="Bronvermelding projecticum introduction | ʚ♡ɞ Oumaima Anbari ʚ♡ɞ" />
<meta property="og:type" content="book" />




<meta name="author" content="Oumaima Anbari" />



<meta name="description" content="Bronvermelding projecticum introduction | ʚ♡ɞ Oumaima Anbari ʚ♡ɞ">

<title>Bronvermelding projecticum introduction | ʚ♡ɞ Oumaima Anbari ʚ♡ɞ</title>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="libs/navigation-1.1/tabsets.js"></script>
<!-- header.html -->
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Eczar:wght@400..800&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Zalando+Sans:ital,wght@0,200..900;1,200..900&display=swap" rel="stylesheet">

<!-- Font Awesome -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/7.0.1/css/all.min.css" integrity="sha512-2SwdPD6INVrV/lHTZbO2nodKhrnDdJK9/kg2XD1r9uGqPo1cUbujc+IYdlYdEErWNu69gVcYgdxlmVmzTWnetw==" crossorigin="anonymous" referrerpolicy="no-referrer" />

<!-- Style Sheet -->
<link rel="stylesheet" href="styles.css" />


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
/* show arrow before summary tag as in bootstrap
TODO: remove if bootstrap in updated in html_document (rmarkdown#1485) */
details > summary {
  display: list-item;
  cursor: pointer;
}
</style>
</head>

<body>

<div class="container-fluid main-container">


<div class="row">
<div class="col-sm-12">
<!--bookdown:toc:end-->
<!--bookdown:toc:start-->
</div>
</div>
<div class="row">
<div class="col-sm-12">
<div id="bronvermelding-projecticum-introduction" class="section level1">
<h1>Bronvermelding projecticum introduction</h1>
<nav class="social">
<div>
<p><i class="fa-brands fa-github"></i>
<a class="darkerText" href="https://github.com/oumaimaanbari" target="_blank">Github </a></p>
</div>
<div>
<p><i class="fa-brands fa-linkedin"></i>
<a class="darkerText" href="https://www.linkedin.com/in/oumaima-anbari-7ab33014a/" target="_blank">LinkedIn </a></p>
</div>
</nav>
<nav class="navBar">
<hr>
<ul>
<li>
<a href="index.html">HOME</a>
</li>
<li>
<a href="section.html">CURRICULUM VITAE</a>
</li>
<li>
<a href="section.html">PORTFOLIO     ▾</a>
<ul class="dropdown">
<li>
<a href="vrije-ruimte.html">Vrije Ruimte</a>
</li>
<li>
<a href="data-management.html">Data Management</a>
</li>
<li>
<a href="reproduceerbaarheid.html">Reproduced Data</a>
</li>
<li>
<a href="bronvermelding-projecticum-introduction.html">Research</a>
</li>
<li>
<a href="parametric-report-met-rmarkdown.html">Parametric Report</a>
</li>
<li>
<a href="custom-r-tools.html">Custom R tools</a>
</li>
</ul>
</li>
</ul>
<hr>
</nav>
<p><br>
<br>
<br></p>
<div id="introductie" class="section level2 unnumbered">
<h2>Introductie</h2>
<p>RNA-sequencing (RNA-seq) is een techniek om genexpressieprofielen van cellen in kaart te brengen, en speelt een belangrijke rol in het begrijpen van ziekten zoals kanker. Dankzij de opkomst van longread sequencing technologie, zoals de minION van Oxford Nanopore, kunnen volledige RNA transcripten in één keer worden gelezen. Dit maakt het mogelijk om complexe genstructuren en alternative splicing nauwkeuriger te analyseren <span class="citation">(<a href="#ref-jainOxfordNanoporeMinION2016">Jain et al. 2016</a>)</span>.</p>
<p>In dit project is RNA-sequencing uitgevoerd op de borstkankercellijn MCF7 met behulp van de MinION-sequencer, waarna een genexpressieanalyse is gedaan. Het doel was om de verkregen data uiteindelijk te vergelijken met een gepubliceerde dataset van Chen et al. <span class="citation">(<a href="#ref-chenSystematicBenchmarkNanopore2025">2025</a>)</span>, waarin MCF7 expressieprofielen op een vergelijkbare methode zijn verkregen. Daarbij zijn onder andere principal component analysis (PCA) en genexpressiecorrelaties gebruikt om de datasets te vergelijken.</p>
<p>Hoewel het oorspronkelijke plan was om gebruik te maken van unique molecular identifiers (UMIs) om PCR-duplicaten te verwijderen en zo de data te verbeteren, is deze stap uiteindelijk niet uitgevoerd. Toch blijft het gebruik van UMIs een waardevolle aanpak binnen RNA-seq, omdat het zorgt voor een betrouwbaardere kwantificatie van de transcripten en dus uiteindelijk de genexpressie resultaten <span class="citation">(<a href="#ref-zhengMassivelyParallelDigital2017">Zheng et al. 2017</a>)</span>.</p>
<p>Nanoplastics zijn een opkomende risicofactor binnen de milieutoxicologie en kankerbiologie. Recente studies suggereren dat blootstelling aan nanoplastics kan leiden tot veranderingen in genexpressie door cel stress, apoptose en metaalionhomeostase <span class="citation">(<a href="#ref-tucerTranscriptomicImplicationsToxic2023">Tucer et al. 2023</a>)</span>.</p>
<p>Het belang van sequencingtechnologie zoals RNA-seq en UMIs in kankeronderzoek wordt steeds groter. Zo tonen bijvoorbeeld Bewicke-Copley et al. <span class="citation">(<a href="#ref-bewicke-copleyApplicationsAnalysisTargeted2019">2019</a>)</span> aan dat sequencing met UMIs het mogelijk maakt om genexpressie met hoge nauwkeurigheid te meten en zo beter inzicht te krijgen in tumorbiologie en de reactie van kankercellen op behandelingen.</p>
<p>Dit benadrukt het belang van technieken zoals UMIs in de betrouwbaarheid en nauwkeurigheid van sequencingdata, wat ook relevant is voor de interpretatie van de differentiële genexpressie in dit project.
<br>
<br>
<br></p>
<hr>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-bewicke-copleyApplicationsAnalysisTargeted2019" class="csl-entry">
Bewicke-Copley, Findlay, Emil Arjun Kumar, Giuseppe Palladino, Koorosh Korfi, and Jun Wang. 2019. <span>“Applications and Analysis of Targeted Genomic Sequencing in Cancer Studies.”</span> <em>Computational and Structural Biotechnology Journal</em> 17: 1348–59. <a href="https://doi.org/10.1016/j.csbj.2019.10.004">https://doi.org/10.1016/j.csbj.2019.10.004</a>.
</div>
<div id="ref-chenSystematicBenchmarkNanopore2025" class="csl-entry">
Chen, Ying, Nadia M. Davidson, Yuk Kei Wan, Fei Yao, Yan Su, Hasindu Gamaarachchi, Andre Sim, et al. 2025. <span>“A Systematic Benchmark of <span>Nanopore</span> Long-Read <span>RNA</span> Sequencing for Transcript-Level Analysis in Human Cell Lines.”</span> <em>Nature Methods</em> 22 (4): 801–12. <a href="https://doi.org/10.1038/s41592-025-02623-4">https://doi.org/10.1038/s41592-025-02623-4</a>.
</div>
<div id="ref-jainOxfordNanoporeMinION2016" class="csl-entry">
Jain, Miten, Hugh E. Olsen, Benedict Paten, and Mark Akeson. 2016. <span>“The <span>Oxford Nanopore MinION</span>: Delivery of Nanopore Sequencing to the Genomics Community.”</span> <em>Genome Biology</em> 17 (1): 239. <a href="https://doi.org/10.1186/s13059-016-1103-0">https://doi.org/10.1186/s13059-016-1103-0</a>.
</div>
<div id="ref-tucerTranscriptomicImplicationsToxic2023" class="csl-entry">
Tucer, Merve, Nazlıcan Tunç, Suat Tucer, and Urartu Seker. 2023. <em>Transcriptomic <span>Implications</span> of <span>Toxic Effects</span> of <span>Nanoparticles</span> on <span>Metabolic Pathways</span> of <span>Liver Cells</span></em>. <a href="https://doi.org/10.1101/2023.11.24.568576">https://doi.org/10.1101/2023.11.24.568576</a>.
</div>
<div id="ref-zhengMassivelyParallelDigital2017" class="csl-entry">
Zheng, Grace X. Y., Jessica M. Terry, Phillip Belgrader, Paul Ryvkin, Zachary W. Bent, Ryan Wilson, Solongo B. Ziraldo, et al. 2017. <span>“Massively Parallel Digital Transcriptional Profiling of Single Cells.”</span> <em>Nature Communications</em> 8 (January): 14049. <a href="https://doi.org/10.1038/ncomms14049">https://doi.org/10.1038/ncomms14049</a>.
</div>
</div>
<p style="text-align: center;">
<a href="custom-r-tools.html"><button class="btn btn-default">Previous</button></a>
<a href="references.html"><button class="btn btn-default">Next</button></a>
</p>
</div>
</div>


</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});

</script>

</body>
</html>
