<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta name="viewport" content="width=device-width, initial-scale=1" />
<meta property="og:title" content="Vrije ruimte | ʚ♡ɞ Oumaima Anbari ʚ♡ɞ" />
<meta property="og:type" content="book" />




<meta name="author" content="Oumaima Anbari" />



<meta name="description" content="Vrije ruimte | ʚ♡ɞ Oumaima Anbari ʚ♡ɞ">

<title>Vrije ruimte | ʚ♡ɞ Oumaima Anbari ʚ♡ɞ</title>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="libs/navigation-1.1/tabsets.js"></script>
<!-- header.html -->
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Eczar:wght@400..800&display=swap" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Zalando+Sans:ital,wght@0,200..900;1,200..900&display=swap" rel="stylesheet">

<!-- Font Awesome -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/7.0.1/css/all.min.css" integrity="sha512-2SwdPD6INVrV/lHTZbO2nodKhrnDdJK9/kg2XD1r9uGqPo1cUbujc+IYdlYdEErWNu69gVcYgdxlmVmzTWnetw==" crossorigin="anonymous" referrerpolicy="no-referrer" />

<!-- Style Sheet -->
<link rel="stylesheet" href="styles.css" />


<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>


<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
/* show arrow before summary tag as in bootstrap
TODO: remove if bootstrap in updated in html_document (rmarkdown#1485) */
details > summary {
  display: list-item;
  cursor: pointer;
}
</style>
</head>

<body>

<div class="container-fluid main-container">


<div class="row">
<div class="col-sm-12">
<!--bookdown:toc:end-->
<!--bookdown:toc:start-->
</div>
</div>
<div class="row">
<div class="col-sm-12">
<div id="vrije-ruimte" class="section level1">
<h1>Vrije ruimte</h1>
<nav class="social">
<div>
<p><i class="fa-brands fa-github"></i>
<a class="darkerText" href="https://github.com/oumaimaanbari" target="_blank">Github </a></p>
</div>
<div>
<p><i class="fa-brands fa-linkedin"></i>
<a class="darkerText" href="https://www.linkedin.com/in/oumaima-anbari-7ab33014a/" target="_blank">LinkedIn </a></p>
</div>
</nav>
<nav class="navBar">
<hr>
<ul>
<li>
<a href="index.html">HOME</a>
</li>
<li>
<a href="section.html">CURRICULUM VITAE</a>
</li>
<li>
<a href="section.html">PORTFOLIO     ▾</a>
<ul class="dropdown">
<li>
<a href="vrije-ruimte.html">Vrije Ruimte</a>
</li>
<li>
<a href="data-management.html">Data Management</a>
</li>
<li>
<a href="reproduceerbaarheid.html">Reproduced Data</a>
</li>
<li>
<a href="bronvermelding-projecticum-introduction.html">Research</a>
</li>
<li>
<a href="parametric-report-met-rmarkdown.html">Parametric Report</a>
</li>
<li>
<a href="custom-r-tools.html">Custom R tools</a>
</li>
</ul>
</li>
</ul>
<hr>
</nav>
<p><br>
<br>
<br></p>
<div id="tijdsbesteding-vrije-opdracht" class="section level2">
<h2>Tijdsbesteding vrije opdracht</h2>
<table>
<thead>
<tr class="header">
<th>Activiteit</th>
<th>Uren</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Oriëntatie op mogelijke skills (MR vs scRNA-seq)</td>
<td>6</td>
</tr>
<tr class="even">
<td>Bestuderen tutorials en documentatie</td>
<td>10</td>
</tr>
<tr class="odd">
<td>Analyse-workflow</td>
<td>6</td>
</tr>
<tr class="even">
<td>Experimenteren met R-packages en data</td>
<td>6</td>
</tr>
<tr class="odd">
<td>Uitwerken portfolio en reflectie</td>
<td>4</td>
</tr>
</tbody>
</table>
</div>
<div id="single-cell-rna-sequencing" class="section level2">
<h2>Single Cell RNA sequencing</h2>
<p>Single cell RNA sequencing (scRNA-seq) is een techniek waarmee de genexpressie van individuele cellen wordt gemeten. In tegestelling tot bulk RNA sequencing, waarbij de gemiddelde genexpressie van een grote groep ellen tegelijk wordt bepaald. ScRNA-seq maakt het mogelijk om verschillen tussen afzonderlijke cellen zichtbaar te maken.</p>
<p>Bij scRNA-seq wordt RNA uit cellen geïsoleerd en omgezet naar cDNA. Dit kan worden gesequenced en resulteert in een count matrix waarin rijen genen voorstellen en de kolommen de individuele cellen. Per cel staat het aantal reads of Unique Molecular Identifiers (UMIs) per gen.</p>
<p>Het voordeel van scRNA-seq is bijvoorbeeld dat er verschillende celtypen binnen één weefsel onderschieden kunnen worden en verschillen in genexpressie tussen cellen kunnne geanalyseerd worden. ScRNA-seq wordt daarom ook veel toegepast in kankeronderzoek.</p>
<div id="workflow" class="section level3">
<h3>Workflow</h3>
<p>In de scRNA-seq analysen worden grote datasets gegenereert, Een standaard workflow is als volgt:</p>
<ol style="list-style-type: decimal">
<li>Inlezen raw count matrix</li>
<li>Kwaliteitscontrole (QC) per cel</li>
<li>Normalisatie van genexpressiedata</li>
<li>Dimensie reduction met bijvoorbeeld een PCA of UMAP</li>
<li>Clustering van cellen</li>
<li>Identificatie van markergenen</li>
</ol>
<p>Veel gebruike R packages voor het standaardiseren van de scRNA-seq analyse zijn Seurat, SingleCellExperiment en scran.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="vrije-ruimte.html#cb1-1" tabindex="-1"></a>raw_mtx <span class="ot">&lt;-</span> <span class="fu">readMM</span>(<span class="st">&#39;counts_unfiltered/cellranger/matrix.mtx&#39;</span>) </span>
<span id="cb1-2"><a href="vrije-ruimte.html#cb1-2" tabindex="-1"></a></span>
<span id="cb1-3"><a href="vrije-ruimte.html#cb1-3" tabindex="-1"></a>genes <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&#39;counts_unfiltered/cellranger/genes.tsv&#39;</span>, <span class="at">sep =</span> <span class="st">&#39;</span><span class="sc">\t</span><span class="st">&#39;</span>, <span class="at">header =</span> F)</span>
<span id="cb1-4"><a href="vrije-ruimte.html#cb1-4" tabindex="-1"></a></span>
<span id="cb1-5"><a href="vrije-ruimte.html#cb1-5" tabindex="-1"></a><span class="fu">rownames</span>(raw_mtx) <span class="ot">&lt;-</span> genes[,<span class="dv">1</span>] </span>
<span id="cb1-6"><a href="vrije-ruimte.html#cb1-6" tabindex="-1"></a></span>
<span id="cb1-7"><a href="vrije-ruimte.html#cb1-7" tabindex="-1"></a><span class="fu">colnames</span>(raw_mtx) <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">&#39;counts_unfiltered/cellranger/barcodes.tsv&#39;</span>, <span class="at">sep =</span> <span class="st">&#39;</span><span class="sc">\t</span><span class="st">&#39;</span>, <span class="at">header =</span> F)[,<span class="dv">1</span>]</span>
<span id="cb1-8"><a href="vrije-ruimte.html#cb1-8" tabindex="-1"></a></span>
<span id="cb1-9"><a href="vrije-ruimte.html#cb1-9" tabindex="-1"></a><span class="co"># Droplets</span></span>
<span id="cb1-10"><a href="vrije-ruimte.html#cb1-10" tabindex="-1"></a><span class="do">## Kansberekening dat de barcode een cel is</span></span>
<span id="cb1-11"><a href="vrije-ruimte.html#cb1-11" tabindex="-1"></a>out <span class="ot">&lt;-</span> <span class="fu">emptyDrops</span>(raw_mtx) </span>
<span id="cb1-12"><a href="vrije-ruimte.html#cb1-12" tabindex="-1"></a><span class="co"># Standaard minimum voor de kans filteren</span></span>
<span id="cb1-13"><a href="vrije-ruimte.html#cb1-13" tabindex="-1"></a>keep <span class="ot">&lt;-</span> out<span class="sc">$</span>FDR <span class="sc">&lt;=</span> <span class="fl">0.05</span> </span>
<span id="cb1-14"><a href="vrije-ruimte.html#cb1-14" tabindex="-1"></a><span class="co"># Lege drops verwijderen</span></span>
<span id="cb1-15"><a href="vrije-ruimte.html#cb1-15" tabindex="-1"></a>keep[<span class="fu">is.na</span>(keep)] <span class="ot">&lt;-</span> <span class="cn">FALSE</span></span>
<span id="cb1-16"><a href="vrije-ruimte.html#cb1-16" tabindex="-1"></a>filt_mtx <span class="ot">&lt;-</span> raw_mtx[,keep] </span>
<span id="cb1-17"><a href="vrije-ruimte.html#cb1-17" tabindex="-1"></a></span>
<span id="cb1-18"><a href="vrije-ruimte.html#cb1-18" tabindex="-1"></a><span class="co"># QC</span></span>
<span id="cb1-19"><a href="vrije-ruimte.html#cb1-19" tabindex="-1"></a><span class="do">## Aantal counts/UMI per cel</span></span>
<span id="cb1-20"><a href="vrije-ruimte.html#cb1-20" tabindex="-1"></a>nCount_rna <span class="ot">&lt;-</span> <span class="fu">colSums</span>(filt_mtx)</span>
<span id="cb1-21"><a href="vrije-ruimte.html#cb1-21" tabindex="-1"></a><span class="do">## Aantal genen met expressie per cell</span></span>
<span id="cb1-22"><a href="vrije-ruimte.html#cb1-22" tabindex="-1"></a>nFeature_rna <span class="ot">&lt;-</span> <span class="fu">colSums</span>(filt_mtx <span class="sc">&gt;</span> <span class="dv">0</span>)</span></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb2-1"><a href="vrije-ruimte.html#cb2-1" tabindex="-1"></a>qc <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb2-2"><a href="vrije-ruimte.html#cb2-2" tabindex="-1"></a>  <span class="at">barcode =</span> <span class="fu">colnames</span>(filt_mtx),</span>
<span id="cb2-3"><a href="vrije-ruimte.html#cb2-3" tabindex="-1"></a>  <span class="at">nCount_rna =</span> <span class="fu">as.numeric</span>(nCount_rna),</span>
<span id="cb2-4"><a href="vrije-ruimte.html#cb2-4" tabindex="-1"></a>  <span class="at">nFeature_rna =</span> <span class="fu">as.numeric</span>(nFeature_rna)</span>
<span id="cb2-5"><a href="vrije-ruimte.html#cb2-5" tabindex="-1"></a>)</span>
<span id="cb2-6"><a href="vrije-ruimte.html#cb2-6" tabindex="-1"></a></span>
<span id="cb2-7"><a href="vrije-ruimte.html#cb2-7" tabindex="-1"></a><span class="fu">summary</span>(qc<span class="sc">$</span>nCount_rna)</span></code></pre></div>
<pre><code>##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##     120    5619    7438    8402    9888   60120</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="vrije-ruimte.html#cb4-1" tabindex="-1"></a><span class="fu">summary</span>(qc<span class="sc">$</span>nFeature_rna)</span></code></pre></div>
<pre><code>##    Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
##      13    2088    2538    2645    3148    8095</code></pre>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="vrije-ruimte.html#cb6-1" tabindex="-1"></a><span class="fu">hist</span>(qc<span class="sc">$</span>nCount_rna, <span class="at">breaks =</span> <span class="dv">50</span>,</span>
<span id="cb6-2"><a href="vrije-ruimte.html#cb6-2" tabindex="-1"></a>     <span class="at">main =</span> <span class="st">&quot;UMI counts per cel&quot;</span>, <span class="at">xlab =</span> <span class="st">&quot;nCount_rna&quot;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:unnamed-chunk-3"></span>
<img src="_main_files/figure-html/unnamed-chunk-3-1.png" alt="Verdeling van totale UMI counts (nCount_rna) per cel na filtering." width="672" />
<p class="caption">
Figure 1: Verdeling van totale UMI counts (nCount_rna) per cel na filtering.
</p>
</div>
<p>De verdeling laat zien dat de meeste cellen relatief lage tot middelhoge UMI counts hebben, met een kleinere groep hogere counts.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="vrije-ruimte.html#cb7-1" tabindex="-1"></a><span class="fu">hist</span>(qc<span class="sc">$</span>nFeature_rna, <span class="at">breaks =</span> <span class="dv">50</span>,</span>
<span id="cb7-2"><a href="vrije-ruimte.html#cb7-2" tabindex="-1"></a>     <span class="at">main =</span> <span class="st">&quot;Genes per cel&quot;</span>, <span class="at">xlab =</span> <span class="st">&quot;nFeature_rna&quot;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:unnamed-chunk-4"></span>
<img src="_main_files/figure-html/unnamed-chunk-4-1.png" alt="Verdeling van het aantal gedetecteerde genen per cel (nFeature_rna) na filtering." width="672" />
<p class="caption">
Figure 2: Verdeling van het aantal gedetecteerde genen per cel (nFeature_rna) na filtering.
</p>
</div>
<p>De meeste cellen vertonen een vergelijkbaar aantal gedetecteerde genen, wat wijst op consistente datakwaliteit na filtering.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb8-1"><a href="vrije-ruimte.html#cb8-1" tabindex="-1"></a><span class="fu">plot</span>(qc<span class="sc">$</span>nCount_rna, qc<span class="sc">$</span>nFeature_rna,</span>
<span id="cb8-2"><a href="vrije-ruimte.html#cb8-2" tabindex="-1"></a>     <span class="at">pch =</span> <span class="dv">16</span>, <span class="at">cex =</span> <span class="fl">0.3</span>,</span>
<span id="cb8-3"><a href="vrije-ruimte.html#cb8-3" tabindex="-1"></a>     <span class="at">xlab =</span> <span class="st">&quot;nCount_rna&quot;</span>, <span class="at">ylab =</span> <span class="st">&quot;nFeature_rna&quot;</span>,</span>
<span id="cb8-4"><a href="vrije-ruimte.html#cb8-4" tabindex="-1"></a>     <span class="at">main =</span> <span class="st">&quot;QC plot van count versus genes per cel&quot;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:unnamed-chunk-5"></span>
<img src="_main_files/figure-html/unnamed-chunk-5-1.png" alt="QC scatterplot na filtering: relatie tussen totale UMI counts (nCount_rna) en het aantal gedetecteerde genen (nFeature_rna) per cel." width="672" />
<p class="caption">
Figure 3: QC scatterplot na filtering: relatie tussen totale UMI counts (nCount_rna) en het aantal gedetecteerde genen (nFeature_rna) per cel.
</p>
</div>
<p>Elke punt is één cel. Cellen met meer UMI counts hebben meer gedetecteerde genen, wat past bij biologisch data na het verwijderen van lege droplets.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="vrije-ruimte.html#cb9-1" tabindex="-1"></a><span class="co"># Dimensions check</span></span>
<span id="cb9-2"><a href="vrije-ruimte.html#cb9-2" tabindex="-1"></a><span class="fu">dim</span>(raw_mtx)</span></code></pre></div>
<pre><code>## [1]  60623 259615</code></pre>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb11-1"><a href="vrije-ruimte.html#cb11-1" tabindex="-1"></a><span class="fu">dim</span>(filt_mtx)</span></code></pre></div>
<pre><code>## [1] 60623  1227</code></pre>
<p>De originele count matrix bestond uit <strong>60.623 genen</strong> en <strong>259.615 barcodes</strong>.<br />
Na filtering bleven <strong>1.227 cellen</strong> over, wat wijst op het verwijderen van lege droplets/low-signal barcodes.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="vrije-ruimte.html#cb13-1" tabindex="-1"></a><span class="co"># Normalisatie</span></span>
<span id="cb13-2"><a href="vrije-ruimte.html#cb13-2" tabindex="-1"></a>norm_mtx <span class="ot">&lt;-</span> <span class="fu">t</span>(<span class="fu">t</span>(filt_mtx) <span class="sc">/</span> <span class="fu">colSums</span>(filt_mtx)) <span class="sc">*</span> <span class="fl">1e4</span></span>
<span id="cb13-3"><a href="vrije-ruimte.html#cb13-3" tabindex="-1"></a>log_mtx <span class="ot">&lt;-</span> <span class="fu">log1p</span>(norm_mtx)</span>
<span id="cb13-4"><a href="vrije-ruimte.html#cb13-4" tabindex="-1"></a></span>
<span id="cb13-5"><a href="vrije-ruimte.html#cb13-5" tabindex="-1"></a><span class="co"># Variate van elk gen</span></span>
<span id="cb13-6"><a href="vrije-ruimte.html#cb13-6" tabindex="-1"></a>gen_var <span class="ot">&lt;-</span> <span class="fu">apply</span>(log_mtx, <span class="dv">1</span>, var)</span>
<span id="cb13-7"><a href="vrije-ruimte.html#cb13-7" tabindex="-1"></a></span>
<span id="cb13-8"><a href="vrije-ruimte.html#cb13-8" tabindex="-1"></a>var_df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb13-9"><a href="vrije-ruimte.html#cb13-9" tabindex="-1"></a>  <span class="at">gene =</span> <span class="fu">names</span>(gen_var),</span>
<span id="cb13-10"><a href="vrije-ruimte.html#cb13-10" tabindex="-1"></a>  <span class="at">variance =</span> gen_var</span>
<span id="cb13-11"><a href="vrije-ruimte.html#cb13-11" tabindex="-1"></a>)</span>
<span id="cb13-12"><a href="vrije-ruimte.html#cb13-12" tabindex="-1"></a></span>
<span id="cb13-13"><a href="vrije-ruimte.html#cb13-13" tabindex="-1"></a><span class="fu">hist</span>(var_df<span class="sc">$</span>variance,</span>
<span id="cb13-14"><a href="vrije-ruimte.html#cb13-14" tabindex="-1"></a>     <span class="at">breaks =</span> <span class="dv">100</span>,</span>
<span id="cb13-15"><a href="vrije-ruimte.html#cb13-15" tabindex="-1"></a>     <span class="at">main =</span> <span class="st">&quot;Gen variatie na normalisatie&quot;</span>,</span>
<span id="cb13-16"><a href="vrije-ruimte.html#cb13-16" tabindex="-1"></a>     <span class="at">xlab =</span> <span class="st">&quot;Variantie (log genormaliseerde expressie)&quot;</span>)</span></code></pre></div>
<div class="figure"><span style="display:block;" id="fig:unnamed-chunk-7"></span>
<img src="_main_files/figure-html/unnamed-chunk-7-1.png" alt="Verdeling van gen variatie na normalisatie. Een subset van genen vertoont hoge variatie tussen cellen en is misschien informatief voor downstream analyses." width="672" />
<p class="caption">
Figure 4: Verdeling van gen variatie na normalisatie. Een subset van genen vertoont hoge variatie tussen cellen en is misschien informatief voor downstream analyses.
</p>
</div>
<p>De meeste genen vertonen lage variatie tussen cellen, terwijl een kleinere subset sterk variërende genen laat zien. Deze genen vormen de basis voor dimensiereductie met bijvoorbeeld UMAP en clustering in downstream scRNA-seq analyses.</p>
<p><br>
<br></p>
</div>
</div>
<div id="mendelian-randomisation" class="section level2">
<h2>Mendelian Randomisation</h2>
<p>Mendelian Randomisation (MR) is een statistische methode die wordt gebruikt om causale verbanden te onderzoeken tussen een blootstelling en een uitkomt aan de hand van genetishe varianten. MR baseert zich op het principe dat genetsiche varianten willekeurig worden verdeeld bij de conceptie. Hierdoor kunnen genetische varianten worden gezien als een natuurlijke randomisatie en is de reden dat MR de ´natuurlijke´ randomized control wordt genoemd.</p>
<p>In meeste onderzoeken is het lastig om causale verbanden aan te tonen, omdat resultaten kunnen worden vertekend door confounding factors. MR is hier de oplossing, omdat genetische varianten: vastliggen vòòr ziekteontwikkeling; niet worden beïnvloed door leefstijl en externe omgevingsfactoren; minder gevoelig zijn voor confounding factors.</p>
<div id="workflow-1" class="section level3">
<h3>Workflow</h3>
<p>In de MR analysen worden Genome Wide Association Studie (GWAS) data gebruikt. In R wordt MR vooral uitgevoerd met het package TwoSampleMR.</p>
<ol style="list-style-type: decimal">
<li>Geschikte genetische instrumenten worden geselecteerd</li>
<li>Harmonisatie van blootstelling en uitkomst data</li>
<li>Uitvoeren van MR analyse</li>
<li>Sensitivity analyse en interpretatie</li>
</ol>

</div>
</div>
</div>
<p style="text-align: center;">
<a href="section.html"><button class="btn btn-default">Previous</button></a>
<a href="data-management.html"><button class="btn btn-default">Next</button></a>
</p>
</div>
</div>


</div>

<script>

// add bootstrap table styles to pandoc tables
$(document).ready(function () {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
});

</script>

</body>
</html>
